src/components/Auth/Login.tsx

```diff
@@ -278,6 +262,22 @@ const Login = (props: LoginProps) => {
     return form;
   };
 
+  const handleSubmit = async (e: React.FormEvent) => {
```

Comment:
why move this down? You have done the same thing in the past 😅  When you make a commit, review your PRs before pushing them.